edweiss-firebase/functions/src/functions/todolist/createTodo.ts

```diff
@@ -33,6 +33,13 @@ export const createTodo = onAuthentifiedCall(Functions.createTodo, async (userId
 
     const todoCollection = CollectionOf<Todo>(`users/${userId}/todos`);
 
+    const existingTodos = await todoCollection.where('name', '==', name).get();
+
+    if (!existingTodos.empty) {
```

Comment:
I don't know if this is due to a particular reason behind it but there is no need to write !existingTodos.empty if you are just checking that the existingTodos is not undefined, you have done so in orther parts of the code so maybe it is a little mistake. I would instead just o if(existingTodos)

Reply:
Yes I understand, I didn't consider the existingTodos to be a collection so of course it makes sense that you check if it is empty instead of undefined. Thank you